.shared_rubocop.yml

```diff
@@ -212,6 +212,9 @@ RSpec/FactoryBot/CreateList:
   Exclude:
     - 'spec/dummy/app/services/seeder.rb'
 
+RSpec/FactoryBot/ConsistentParenthesesStyle:
+  Enabled: false
```

Comment:
I like it on, but use the omit style [here](https://docs.rubocop.org/rubocop-rspec/cops_rspec_factorybot.html#enforcedstyle-omit_parentheses)